ChangeLog.md

```diff
@@ -20,6 +20,13 @@ See docs/process.md for more on how version tagging works.
 
 Current Trunk
 -------------
+- Add new setting: `REVERSE_DEPS`. This can be used to control how emscripten
+  decides which reverse dependecies to include.  See `settings.js` for more
+  information.  The default setting ('auto') is the transitional way emscripten
+  has worked in the past so there should be no change unless this options is
+  actually used.  This option partially replaces the `EMCC_ONLY_FORCED_STDLIBS`
+  environment variable which (among other things) essentially had the effect of
+  setting `REVERSE_DEPS` to be 'all'.
```

Comment:
The effect that env var had would depend on the libraries chosen, I think? So it could be the same as `all` or `none`.

Reply:
I think EMCC_ONLY_FORCED_STDLIBS seems to currently/historically mean 'all'.   

See this comment:

```
  # If we are only doing forced stdlibs, then we don't know the actual                               
  # symbols we need, and must assume all of deps_info must be exported.                              
  # Note that this might cause warnings on exports that do not exist.   
```

Note the *all* in "and must assume all of deps_info".  So whether is a good idea or not, that is what it seems to mean.  Hopefully we can get rid of it some point.

Reply:
Does that make sense?  OK to land now?

Reply:
I see, yes. lgtm. I guess that old option was slightly confusing.